native/src/utils/DatabaseConnector.ts

```diff
@@ -510,6 +517,34 @@ class DatabaseConnector {
     return this.readFile(path, mapPoisJson)
   }
 
+  async storeLocalNews(localNews: LocalNewsModel[], context: DatabaseContext): Promise<void> {
+    const jsonModels = localNews.map(
+      (it: LocalNewsModel): ContentLocalNewsJsonType => ({
```

Comment:
You don't need to set the type of the `localNews` again, it's already set in the argument typing two lines earlier.